### PR TITLE
Basic issue report template

### LINKED ISSUE #233: Bug report template
Can you please add bug report template?
Template from this repo https://github.com/ValveSoftware/steam-for-linux or this https://github.com/ValveSoftware/Proton
can be used as base

### PR DESCRIPTION
**Pull request by kisak-valve**
Basic issue report template.

Closes #233.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: Bug report
+about: Issues with the Steam runtime
+
+---
+
+#### Your system information
+
+* Steam Runtime Version: 
+* Distribution (e.g. Ubuntu): 
+* Link to your full system information in a [Gist](https://gist.github.com/):
```
Comment (smcv) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]: Perhaps
```suggestion
* Link to your full system information (Help -> System Information) in a [Gist](https://gist.github.com/): 
```

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+#### Your system information
+
+* Steam Runtime Version: 
+* Distribution (e.g. Ubuntu): 
+* Link to your full system information in a [Gist](https://gist.github.com/): 
+  <!-- Please wait for the extended system infomation to be collected by Steam -->
+* Have you checked for system updates?: [Yes/No]
+
```
Comment (smcv) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]:
```suggestion
* Are you using the experimental SteamLinuxRuntime compatibility tool?: [Yes/No]

```

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Issues with the Steam runtime
+
+---
+
+#### Your system information
+
+* Steam Runtime Version: 
+* Distribution (e.g. Ubuntu):
```
Comment (smcv) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]:
```suggestion
* Distribution (e.g. Ubuntu 18.04): 
```

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (kisak-valve) [reply .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]: I adjusted this a little bit when adding it to the second iteration. Does that work for you?

**Pull request merged**